experimental/dds/tree2/src/core/tree/types.ts

```diff
@@ -161,3 +161,10 @@ export interface NodeData {
 	 */
 	readonly type: TreeNodeSchemaIdentifier;
 }
+
+/**
+ * When creating a node which sits above the detached fields, and thus is not a real node and who's type should not matter, use this type to indicate that.
```

Comment:
Nit (phrasing is a bit confusing). Maybe something like:
```suggestion
 * Use this type to indicate that a node sits above the detached fields, and thus is not a real node and who's type should not matter.
```